### PR TITLE
(PCP-374) Update Catch configuration

### PR DESCRIPTION
**Pull request by parisiale**
Updating tests/main.cc to Catch v1.4.0 changes.

### CODE REVIEW

**Comment by parisiale** [comment]
Don't merge, as we need a new leatherman release.

**Comment by parisiale** [comment]
Waiting for https://github.com/puppetlabs/cpp-pcp-client/pull/148

**Comment by parisiale** [comment]
This can now be merged.

File: lib/tests/main.cc [hunk 1]
```diff
@@ -4,31 +4,40 @@
 
 #include <catch.hpp>
 
+#include <boost/nowide/iostream.hpp>
```
Comment (mruzicka) [inline lib/tests/main.cc]: What is this needed for?

**Comment by mruzicka** [comment]
LGTM

File: lib/tests/main.cc (hunk 1 above)
Comment (parisiale) [inline lib/tests/main.cc]: It's not needed; it got in by mistake for some i18n tests in another branch.

BTW, when you asked about this in the channel I thought you meant the configuration.cc one; please let's keep code conversation here.

**Pull request merged**